src/Cubes/Cubes.jl

```diff
@@ -510,7 +510,11 @@ function DD.show_after(io::IO, mime, c::YAXArray)
     blockwidth = get(io, :blockwidth, 0)
     DD.print_block_separator(io, "file size", blockwidth, blockwidth)
     println(io, " ")
-    println(io, "  file size: ", formatbytes(cubesize(c)))
+    try
+        println(io, "  file size: ", formatbytes(cubesize(c)))
+    catch e
+        e isa ErrorException ? println(" could not determine DataType size.") : rethrow(e)
```

Comment:
If we rethrow the error, we will get a better message but still can't show the cube without an error. So I think that this should not be rethrowing.

Reply:
@felixcremer what's is blocking this?